### ISSUE TITLE
"donejs add app <app-name>" generates application directory but puts the app files in the parent directory

### ISSUE THREAD

**Issue by mjstahl** [donejs/generator-donejs#282]
__How often can you reproduce it?__ 

- [X] Always
- [ ] Sometimes
- [ ] Rarely
- [ ] Unable
- [ ] I didn’t try

__Description:__
When running `donejs add app APP_NAME` from a given directory the `APP_NAME` directory is created and contains ONLY a `node_modules` directory and a `package-lock.json` file.  ALL of the application files exist in the parent directory (the directory the `donejs` command was executed from).

This also applies to creating the APP_NAME directory first, changing into that directory, and executing `donejs add app`.  The contents of the APP_NAME directory will be `node_modules` and `package-lock.json`, the parent directory will contain the application files.

__Steps to reproduce:__

1. `mkdir TEST`
2. `cd TEST`
3. `donejs add app generator-fail --yes`
4. `ls` (observe the application files in the TEST directory)
5. `cd generator-fail` (observe only `node_modules` and `package-lock.json` in app directory)
6. (in the TEST directory) `rm -rf *`
7. `mkdir generator-fail`
8. `cd generator-fail`
9. `donejs add app --yes`
10. `ls` (observe only `node_modules` and `package-lock.json` in current directory)
11. `ls ..` (observe the application files in the TEST directory)

__Expected results:__
All of the generated files should be contained within the generated application directory.

__Actual results:__
Refer to the steps to reproduce.

__Environment:__

| Software                             | Version
| -------------------------- | -------
| donejs -V                            | 2.1.0
| npm ls generator-donejs   | 2.1.0
| node -v                               | 9.8.0
| npm -v                                | 5.7.1
| Operating system              | macOS 10.13.3

**Comment by justinbmeyer** [comment]
If you don’t have —yes, does it work? I did this 2 days ago without —yes.

**Comment by mjstahl** [comment]
It does not work with or without `--yes`

It also fails when creating a component (`donejs add component`)

```js
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Expected to find a package.json file at /Users/mark/Projects/WeMeetBetter/package.json but did not
    at Object.exports.getPkgOrBail (/Users/mark/Projects/WeMeetBetter/client-www/node_modules/generator-donejs/lib/utils.js:48:19)
```

**Comment by mjstahl** [comment]
Yes, I have tried it with and without `--yes` and it fails per the issue in both cases.

**Comment by matthewp** [comment]
I wasn't able to recreate given those steps. Is it possible that you had some files in the `TEST` folder already? If so what were they?

**Comment by matthewp** [comment]
pinging @mjstahl
